Urbuddi/serenity.properties

```diff
@@ -9,8 +9,8 @@ serenity.take.screenshots = BEFORE_AND_AFTER_EACH_STEP
 serenity.outputDirectory = target/site/serenity/screenshots
 webdriver.autodownload = true
 
-admin.username=automation@gmail.com
-admin.password=automation@123
+admin.username=automationadminone@yopmail.com
```

Comment:
@aramya2021 we should not send the credentials to git. check best practice while pushing the code